openshift/4.0/templates/sonarqube/deploy.yaml

```diff
@@ -1,163 +0,0 @@
-apiVersion: v1
```

Comment:
Defines a set of objects needed to deploy and run a SonarQube application.

Created:
	ImageStream - sonarqube
	DeploymentConfig - sonarqube
	PersistentVolumeClaim
	Service - sonarqube
	Route - sonarqube